### ISSUE TITLE
Unicode error with utf-8 encoding value

### ISSUE THREAD

**Issue by gaellbn** [Azure/azure-sdk-for-python#273]
Hi,

i've some problem with not decoding utf-8 char

```python
  File "/Users/gprudhomme/Documents/Missions/cloukr/temp/testlibcloud/lib/python2.7/site-packages/azure/__init__.py", line 608, in _get_node_value
    return data_type(value)
UnicodeEncodeError: 'ascii' codec can't encode character u'\xe9' in position 6: ordinal not in range(128)
```
xml content 
```xml
<SubnetNames><SubnetName>LBN_Id\xc3\xa9fix-Spare</SubnetName></SubnetNames>
```

**Comment by huguesv** [comment]
Fixed in upcoming v0.10 release.

**Comment by gaellbn** [comment]
thanks @huguesv .
